lib/main.dart

```diff
@@ -13,8 +15,9 @@ import 'package:bien_aca_quarantine/components/pages/alert_page_out_of_zone.dart
 import 'package:bien_aca_quarantine/constants/BienAcaConstants.dart';
 import 'package:bien_aca_quarantine/services/NavigationService.dart';
 
-void main() {
+void main() async {
```

Comment:
Are you sure this won't mess the initialization? Can we just not await `DotEnv().load`?

I'm asking because I saw some delays with black screen when `main` is `async`, but maybe the stuff I was doing was too complex and this is pretty simple afaict.

Reply:
I had the same doubt too, so I researched a little bit about it. It seems that it doesn't really matter if the function is async or not, but if it contains a promise and it can be permitted not to be awaited. It technically would work both ways, but as we're loading `env` variables here, I choose to await it, just in case.